ocaml/lambda/translmod.ml

```diff
@@ -45,8 +45,16 @@ exception Error of Location.t * error
 
 (* CR layouts v2: This is used as part of the "void safety check" in the case of
    `Tstr_eval`, where we want to allow `any` in particular.  Remove when we
-   remove the safety check. *)
+   remove the safety check. But we still default to value before checking for
+   void, to allow for sort variables arising in situations like a module
+   that is just:
+
+     exit 1;;
+
+   When this sanity check is removed, consider whether it must be replaced with
+   some defaulting. *)
 let layout_must_not_be_void loc ty layout =
```

Comment:
Not really part of this patch, but this function is duplicated with translcore. Maybe move to Layout?

Reply:
#1402 changes `translmod` to do something different here, so it's no longer duplicated.  I agree it would be cleaner to move all the functions, but they are an artifact of the void safety check so I'd rather wait until we eliminate that to consider what's the best thing.